src/mantarray_desktop_app/utils.py

```diff
@@ -86,22 +86,21 @@ def validate_customer_credentials(request_args: Dict[str, Any], shared_values_di
         request_args: dictionary containing the new user configuration settings.
         shared_values_dict: dictionary containing stored customer settings.
     """
-    customer_account_uuid = request_args.get("customer_account_uuid", None)
+    customer_account_id = request_args.get("customer_account_uuid", None)
     customer_pass_key = request_args.get("customer_pass_key", None)
-    user_account_id = request_args.get("user_account_id", None)
-    stored_customer_ids = shared_values_dict["stored_customer_settings"]["stored_customer_ids"]
+    stored_customer_id = shared_values_dict["stored_customer_settings"]["stored_customer_id"]
 
-    if customer_account_uuid is not None:
-        if customer_account_uuid in stored_customer_ids:
-            valid_creds = stored_customer_ids[customer_account_uuid]["password"] == customer_pass_key
-            valid_user = user_account_id in stored_customer_ids[customer_account_uuid]["user_account_ids"]
-
-            if not valid_creds:
-                raise InvalidCustomerPasskeyError(customer_pass_key)
-            if not valid_user:
-                raise ImproperlyFormattedUserAccountIDError(user_account_id)
+    if customer_account_id is not None:
+        if stored_customer_id["id"] == customer_account_id:
+            if stored_customer_id["password"] != customer_pass_key:
+                raise InvalidCustomerAccountIDPasswordError()
```

Comment:
first checks if it's already stored in electron, else will call aws route to validate